app/assets/javascripts/application.js

```diff
@@ -19,4 +19,5 @@
 //= require serviceworker-companion
 //= require restrictElements.js
 //= require time.js
+//= require allpages.js
```

Comment:
all js included here will be available in all the pages anyway, So prefer an explicit name like `scroll.js`